modules/acquia_lift_publisher/acquia_lift_publisher.links.task.yml

```diff
@@ -2,3 +2,8 @@ acquia_lift_publisher.entity_config_form:
   route_name: acquia_lift_publisher.entity_config_form
   title: 'Setup'
   base_route: acquia_lift_publisher.entity_config_form
+acquia_lift_publisher.settings_form:
+  route_name: acquia_lift_publisher.general_settings
+  title: 'Settings'
+  weight: 50
+  base_route: acquia_lift_publisher.entity_config_form
```

Comment:
The new general settings form looks really nice, but I think it is an overkill. Also, logically I think it is still a part of the entity configuration.

Let's move the checkbox to the current Setup page, and eliminate the new tab.

Reply:
When I was planning the form I thought about the possibility that it might be a subject of extension in case any other configuration needs to be added. If you think we should move the checkbox to EntityViewModeConfigurationForm anyway I'll do that instead. Should I, in that case as well? Also as the class suggests, it has to do with entity view configurations, although this is more related to general synchronization process. What do you think?

Reply:
I am not expecting any other configuration options at this point. It would be better to introduce the new tab / form only when we know we need. 

I'd recommend that we move the checkbox to EntityViewModeConfigurationForm. I think it can be the first option there. I didn't notice the form name was `EntityViewModeConfigurationForm`. I think it is better to be named `ContentPublishingForm` (or a similar, more precise naming), now.

Reply:
Moved and renamed.